client/my-sites/customer-home/cards/education/earn/payments-features-modal.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from 'react';
+import { BlankCanvas } from 'calypso/components/blank-canvas';
+import VideosUi from 'calypso/components/videos-ui';
+import ModalFooterBar from 'calypso/components/videos-ui/modal-footer-bar';
+import ModalHeaderBar from 'calypso/components/videos-ui/modal-header-bar';
+import { COURSE_SLUGS } from 'calypso/data/courses';
+import '../blogging-quick-start/style.scss';
```

Comment:
This seems a signal that we're duplicating a generic reusable component. What do you think about moving the `blogging-quick-start-modal` component (and its styles) to a shared path and use it for both courses?

Reply:
blogging-quick-start-modal moved do videoUI and renamed to VideoModal. Now it's used both for earn and blogging card.